src/managers/NetworkRequestManager.ts

```diff
@@ -168,23 +184,41 @@ class NetworkRequestManager {
 		script: string | undefined,
 		requestId: string,
 		response?: EndpointResponse | undefined,
+		auditInfo?: {
+			log: AuditLog;
+			scriptType: Exclude<RequestEvent['eventType'], 'request'>;
+			associatedId: string;
+		},
 	): Promise<string | undefined> {
 		if (script && script != '') {
 			try {
-				const sprocketPan = getScriptInjectionCode(requestId, response);
+				auditInfo &&
```

Comment:
you asked for nitpick, so nitpick is mixing && shortcut syntax with if statements in a single file/codebase/section is pretty confusing imo. Is there anything wrong with checking auditinfo using an if statement?